src/css/index.scss

```diff
@@ -1,79 +0,0 @@
-/* these style rules are used inside the component shadow root so can reference generic elements without influencing the containing page */
-section {
-  background: #e1f1ee;
-  border-left: 3px solid #006c58;
-  margin: 1rem 0 0 0;
-  padding: 0 2rem 2rem 2rem;
-  display: inline-block;
-  width: 100%;
-  box-sizing: border-box;
-}
-
-h2 {
-  color: #077e62;
-}
-
-ul.benefits {
-  margin: 0;
-  padding: 0;
-  display: flex;
-  flex-direction: row;
-  flex-wrap: wrap;
-  gap: 1rem;
-
-  li {
-    list-style: none;
-    background: #fff;
-    flex-grow: 1;
-    max-width: 100%;
-
-    a {
-      display: flex;
-      justify-content: space-between;
-      padding: 1.2rem;
-      text-decoration: none;
-      color: #000;
-
-      &:hover {
-        box-shadow: 0 0 23.3143px rgb(0 0 0 / 25%);
-      }
-    }
-
-    .details {
-      display: flex;
-      min-width: 69%;
-      align-items: center;
-
-      .svg {
-        margin-top: 0.3rem;
-      }
-
```

Comment:
I didn't take a close look but I doubt this CSS needs to be this deeply nested. As a rule I try to keep it to 3 or fewer selectors.